pss/pss.go

```diff
@@ -27,12 +27,21 @@ import (
 // Pss is the top-level struct, which takes care of message sending
 type Pss struct {
 	localStore chunk.Store
+	tag        *chunk.Tag
```

Comment:
i don't understand why `tag` would be an API field, can you please explain?

i thought tags were applied to chunks, it is not clear to me what an API tag value would be

Reply:
This was removed, there must be a tag per TrojanChunk sent, to be able to track it's state.